pkg/operator/sync_kubecontrollermanager_v311_00.go

```diff
@@ -58,12 +67,34 @@ func syncKubeControllerManager_v311_00_to_latest(c KubeControllerManagerOperator
 		errors = append(errors, fmt.Errorf("%q: %v", "sa", err))
 	}
 
+	operatorConfigBytes := v311_00_assets.MustAsset("v3.11.0/kube-controller-manager/operator-config.yaml")
+	operatorConfigObj, err := runtime.Decode(v1alpha1Codecs.UniversalDecoder(v1alpha1.GroupVersion), operatorConfigBytes)
+	if err != nil {
+		errors = append(errors, err)
+	}
+	requiredOperatorConfig, ok := operatorConfigObj.(*v1alpha1.KubeControllerManagerOperatorConfig)
+	if !ok {
+		errors = append(errors, fmt.Errorf("operator-config.yaml is not v1alpha1.KubeControllerManagerOperatorConfig (%t)", operatorConfigObj))
+	}
+	imagePullSpec := os.Getenv("IMAGE")
+	if len(imagePullSpec) == 0 {
+		errors = append(errors, fmt.Errorf("required environment variable IMAGE is not set"))
```

Comment:
don't fail here, just don't overwrite the value if it isn't present and we can fill in a default value in our bindata